### PR TITLE
added market creation methods to Augur.ts

### PR DESCRIPTION
**Pull request by robert-e-davidson3**

### CODE REVIEW

File: packages/augur-sdk/src/api/Market.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    const universe = this.augur.contracts.universe;
+
+    const events = await universe.createCategoricalMarket(
+      params.endTime,
+      params.feePerCashInAttoCash,
+      params.affiliateFeeDivisor,
+      params.designatedReporter,
+      params.outcomes,
+      formatBytes32String(params.topic),
+      params.extraInfo
```
Comment (justinbarry) [inline packages/augur-sdk/src/api/Market.ts:68]: Should we be validating extraInfo here vs expecting a string?

Comment (robert-e-davidson3) [reply packages/augur-sdk/src/api/Market.ts:68]: It is actually just a string. In the contract, it's purely a string that we log. We interpret it as JSON in the UI (and maybe also the middleware?).

**Review by justinbarry — CHANGES_REQUESTED** [review]
One feedback item.

**Review by bthaile — APPROVED** [review]
lgtm, I have code in the UI that converts from display values to onChain values. I'll put that into Augur.ts when the market creation branch gets merged up to master

File: packages/augur-sdk/src/api/Market.ts (hunk 1 above)
Comment (justinbarry) [reply packages/augur-sdk/src/api/Market.ts:68]: It is a string in the contract for event index reasons or some such efficiency thing. They would be first class params if there were room.

Comment (justinbarry) [reply packages/augur-sdk/src/api/Market.ts:68]: I'll file another ticket to get this sorted.

**Review by justinbarry — APPROVED** [review]

File: packages/augur-sdk/src/api/Market.ts (hunk 1 above)
Comment (robert-e-davidson3) [reply packages/augur-sdk/src/api/Market.ts:68]: We can make it an interface then do the json serialization just before the contract call. That will break a ton of tests though so it'd be better in its own ticket.

**Pull request merged**